.github/workflows/amplify_datastore.yaml

```diff
@@ -0,0 +1,26 @@
+name: amplify_datastore
+on:
+  push:
+    branches:
+      - main
+      - stable
+      - next
+  pull_request:
+    paths:
+      - 'packages/amplify_datastore/**/*.dart'
```

Comment:
question: Would it make sense to also include interface paths to this pattern so that changes to the interface execute these tests?

Reply:
Yes, in the future this will be possible to do, specifically when #2068 is merged since it has the semantic understanding of the repo (which packages depend on which).